hack/generated/apis/resources.md

```diff
@@ -43,7 +43,6 @@ v1alpha1api20201101
 - PublicIPAddress
 - VirtualNetwork
 - VirtualNetworkGateway
-- VirtualNetworkTap
```

Comment:
Why remove this resource?

Reply:
See the comment in the PR description:
> I removed VirutalNetworkTap because based on [their documentation](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-tap-overview) you must have the feature enabled on your subscription to even use it and it's still in preview (even though it's in a non-preview API version...)

From what I can tell, the feature looks pretty "dead" to me: Its documentation hasn't been updated since 2019, and there are no samples/examples that use it. The 2019 documentation says that it's in preview and your subscription must opt in and be approved (or something to that effect).